mithril-client-cli/src/commands/cardano_transaction/snapshot_list.rs

```diff
@@ -43,8 +42,7 @@ impl CardanoTransactionSnapshotListCommand {
                 .table()
                 .title(vec![
                     "Epoch".cell(),
-                    "Immutable".cell(),
-                    "Network".cell(),
+                    "Block Number".cell(),
```

Comment:
Do we want to remove the `Network` here?

Reply:
There's no network anymore in the message (btw there's no network for the same reason in the list of MithrilStakeDistribution).

Reply:
I checked and it would be difficult to add back the network in the messages since neither the `SignedEntity<T: Artifact>` nor the CardanoTransactionSnapshot artifact have this property.
We would at least need the adapter to know this property (so build it in the dependency system).

And if this property matters why does it not matter for other signed entities ? If yes we should add it to the `SignedEntity<T>` type.

Reply:
Let's keep it like this for now :ok_hand: